client/beefy/rpc/src/lib.rs

```diff
@@ -62,26 +99,55 @@ pub trait BeefyApi<Notification, Hash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId,
 	) -> jsonrpc_core::Result<bool>;
+
+	/// Returns header of the latest BEEFY finalized block as seen by this client.
+	///
+	/// The latest BEEFY block might not be available if the BEEFY gadget is not running
+	/// in the network or if the client is still initializing or syncing with the network.
+	/// In such case an error would be returned.
+	#[rpc(name = "beefy_getFinalizedBlockHeader")]
```

Comment:
```suggestion
	#[rpc(name = "beefy_getFinalizedHead")]
```
I'd propose that or even `beefy_getHead`. We have `chain_newHeads`, `chain_getHead` and `chain_getFinalizedHead` methods already so I think this would be aligned.